.github/workflows/pypi.yml

```
@@ -3,10 +3,25 @@ on:
     tags:
       - v*
+
 jobs:
-  build-n-publish:
-    uses: fizyk/actions-reuse/.github/workflows/shared-pypi.yml@v4.4.7
-    with:
-      publish: true
-    secrets:
-      pypi_token: ${{ secrets.pypi_token }}
+  build:
+    uses: ./.github/workflows/build.yml
```

actionlint (1.7.12)

[error] error while parsing reusable workflow "./.github/workflows/build.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)